doc/_templates/layout.html

```diff
@@ -30,27 +30,6 @@
     {% endif %}
 {% endblock %}
 
-
-{% block sidebartitle %}
```

Comment:
https://github.com/readthedocs/sphinx_rtd_theme/blob/5a263753d52c1628c88392fbf52c729f5a8e79b5/sphinx_rtd_theme/layout.html#L114-L139

For reference, the above link is the default template for the sidebartitle block in the sphinx_rtd_theme. 

Removing the custom `sidebartitle` block may make the version selector disappear (#3542), so we need to update this block instead of removing it.